src/ext/yuni/src/yuni/io/filename-manipulation.cpp

```diff
@@ -294,18 +294,18 @@ static inline void ExtractFilePathImpl(StringT& out, const AnyString& path, bool
         out.assign(path, pos);
 }
 
-void ExtractFilePath(String& out, const AnyString& path, bool systemDependant)
+void parent_path(String& out, const AnyString& path, bool systemDependant)
 {
-    ExtractFilePathImpl(out, path, systemDependant);
+    parent_path_impl(out, path, systemDependant);
 }
 
-void ExtractFilePath(Clob& out, const AnyString& path, bool systemDependant)
+void parent_path_impl(Clob& out, const AnyString& path, bool systemDependant)
 {
-    ExtractFilePathImpl(out, path, systemDependant);
+    parent_path_impl(out, path, systemDependant);
 }
```

Comment:
I'm not too sure about this. Isn't there a risk of confusion ? The templated function has the same name & number of arguments. We rely on delicate template deduction rules by doing this.

Reply:
I chose to remove this version, since template deduction will do the job. This function must be here for historical reasons, since CTAD only seems to exist since C++17.